x-pack/packages/kbn-cloud-security-posture/src/utils/utils.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
`utils/uitils.ts` naming suggests that you don't need a directory `utils` if you intend to put everything into one file anyway. In general, the better approach is not to have such a "generic" file where there are functions without any structure. It's betterto group cod e by features or by "domain". But in the case of the functions in this file, I would have put each function to where it is being used anyway. I don't see any function reused in multiple places, so no need to move it away from where it is being used